webcompat/static/css/development/components/markdown.css

```diff
@@ -108,12 +96,29 @@
   }
 
   /* pre / code */
+
+  .wc-Markdown code {
+    font-family: monospace;
+    font-size: 90%;
+    white-space: normal;
+  }
+
+  .wc-Markdown pre > code {
```

Comment:
@karlcow the bug is resolved by this rule and only this rule.

Maybe if you want, I can add 2 commit: "clean" and bug resolved"